sortinghat/core/migrations/0010_auto_20210624_1234.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.1 on 2021-06-24 12:34
```

Comment:
This commit should be squashed with the first one. Also, you should try to change the name of the migration. See https://docs.djangoproject.com/en/3.2/ref/django-admin/#cmdoption-makemigrations-name